src/Client.php

```diff
@@ -433,8 +433,8 @@ protected function createPipeline(array $options = null, $callable = null)
             $class = 'Predis\Pipeline\Pipeline';
         }
 
-        /*
-         * @var ClientContextInterface
+        /**
```

Comment:
Alright then, fixed without merging this PR (but put a note in the commit message to thank @Aliance for spotting this one).